Modules/Common/src/ReferenceComparatorCheck.cxx

```diff
@@ -129,6 +133,32 @@ static Quality compare(TCanvas* canvas, ObjectComparatorInterface* comparator, s
   return comparator->compare(plots.first, plots.second, message);
 }
 
+Quality ReferenceComparatorCheck::getSinglePlotQuality(std::shared_ptr<MonitorObject> mo, std::string& message)
+{
+  // retrieve the reference plot and compare
+  auto* th1 = dynamic_cast<TH1*>(mo->getObject());
+  auto referenceRun = std::stoi(mCustomParameters.atOptional("referenceRun").value_or("0"));
```

Comment:
is default value "0" ever worth continuing with? Perhaps we should return `Quality::Null` instead?

Reply:
agreed